src/Model/Donor.php

```diff
@@ -50,10 +52,88 @@ class Donor
      */
     private $mandateSource;
 
-    public function __construct(DonorState $state, string $mandateSource)
-    {
+    /**
+     * @var string
+     */
+    private $payerNumber;
+
+    /**
+     * @var \byrokrat\banking\AccountNumber
```

Comment:
add a `use byrokrat\banking\AccountNumber` statement and this can be rewritten as smiply `@var AccountNumber`